ruby/ql/lib/codeql/ruby/dataflow/internal/DataFlowPublic.qll

```diff
@@ -314,6 +317,34 @@ module Content {
   class UnknownPairValueContent extends PairValueContent, TUnknownPairValueContent {
     override string toString() { result = "pair" }
   }
+
+  /**
```

Comment:
I'm not sure which class you're referring to
- `TOptionalContentSet` is entirely hidden in `DataFlowPrivate`.
- `OptionalTypeTrackerContent` is defined in `TypeTrackerSpecific.qll`
- `Content::Attribute` is not hidden as it can be wrapped in a `ContentSet`. We could hide the class, but the values can still leak out to the user.